satpy/tests/reader_tests/test_seadas_l2.py

```diff
@@ -118,6 +118,85 @@ def _add_variable_to_file(h, var_name, var_info):
         setattr(v, attr_key, attr_val)
 
 
+@pytest.fixture(scope="module")
+def seadas_l2_modis_chlor_a_netcdf(tmp_path_factory):
+    """Create MODIS SEADAS NetCDF file."""
+    filename = "t1.21332.1758.seadas.nc"
+    full_path = str(tmp_path_factory.mktemp("seadas_l2") / filename)
+    return _create_seadas_chlor_a_netcdf_file(full_path, "Terra", "MODIS")
+
+
+def _create_seadas_chlor_a_netcdf_file(full_path, mission, sensor):
+    from netCDF4 import Dataset
+    nc = Dataset(full_path, "w")
+    nc.createDimension("number_of_lines", 5)
+    nc.createDimension("pixels_per_line", 5)
+    nc.instrument = sensor
+    nc.platform = mission
+    nc.time_coverage_start = "2021-11-18T17:58:53.191Z"
+    nc.time_coverage_end = "2021-11-18T18:05:51.214Z"
+
+    lon_info = {
```

Comment:
Could it be cleaner to factorise the creation of the variables for hdf and nc?

Reply:
I thought about it, but there were enough differences in attribute names and attributes existing or not existing that it didn't seem worth it. Like I could make an xarray Dataset or a dictionary of dictionaries and use that to write stuff out, but I wanted to be sure of data types being used.